src/routes/MapExplorer/MapExplorer.js

```diff
@@ -139,7 +138,8 @@ export default class MapExplorer extends React.Component {
         zoom={zoom}
       />
       <button styleName={cx('toggleDrawerButton', { 'drawerOpen': showDrawer })} onClick={this.toggleDrawer}>
-        <Icon name='Stack' green={showDrawer} styleName='icon' />
+        <img styleName={cx('openDrawer')} src='/icon/menu.svg' />
```

Comment:
dont need to use cx if its just one class, its just styleName='openDrawer', but also we should replace these with <Icon>s